teest_ml_ready.py

```diff
@@ -0,0 +1,9 @@
+import resdk
+from resdk.tables import MLTables
+
+res = resdk.Resolwe(url='https://qa.genialis.io')
+
+c = res.collection.get("reference-spaces")
+mt = MLTables(c, name="Test 1.2.3")
+
+print(mt.exp)
```

Comment:
Yes. Will remove before the merge, but I'm keeping it as my dirty way of checking the functionality + it might come handy for someone doing fast PR review trying to test the functionality.